src/test/java/uk/gov/hmcts/reform/jobscheduler/controllers/jobscontroller/GetAllTest.java

```diff
@@ -34,35 +40,67 @@
     @MockBean private AuthService authService;
 
     @Test
-    public void should_return_no_data_when_empty_job_list_is_returned() throws Exception {
-        String emptyResponse = "{\"data\":[]}";
+    public void should_return_no_data_when_empty_job_list_is_returned() {
+        ResultMatcher contentIsArray = jsonPath("content").isArray();
+        ResultMatcher contentIsEmpty = jsonPath("content").isEmpty();
+        ResultMatcher pageIsLast = jsonPath("last").value(true);
+        ResultMatcher pageIsFirst = jsonPath("first").value(true);
```

Comment:
Those are now used once and can be inlined.

Reply:
yep 😇